es6-lib/util/metrics.js

```diff
@@ -9,12 +9,59 @@ var conf = config();
 
 class Metrics {
   constructor() {
-    setInterval(this._sampleMemory.bind(this), 1000);
+    this._resetState();
     this._gc = gcstats();
     this._gc.on('stats', _.throttle(this._onGcStat, 1000).bind(this));
   }
 
-  static heapdump(req, res) {
+  _resetState() {
+    this._state = {
+      http: {
+        status: {}
+      },
+      gc: {},
+      memory: {}
+    };
+  }
+
+  metrics(req, res) {
+    res.status(200).send(JSON.stringify(this.state));
+    this._resetState();
```

Comment:
my understanding is that collectd handles that. they're reset each time `/metrics` is called. it looks like collectd will divide the count by the time between samples in order to get the rate; https://collectd.org/wiki/index.php/Data_source